datumaro/plugins/coco_format/extractor.py

```diff
@@ -100,6 +109,34 @@ def _load_person_kp_categories(self, loader):
 
         return categories
 
+    @staticmethod
+    def _load_panoptic_config(path):
+        with open(path, 'r') as f:
+            import json
+            return json.load(f)
+
+    def _load_panoptic_categories(self, config, root_path):
```

Comment:
From https://cocodataset.org/#format-data I understood that the panoptic task has the same json structure as others and includes color info. Why the current solution uses a separate file?

Reply:
Because it's not actually the same, but yes. quite similar. If you will try to open it with pycocotools, you'll get the same parse error https://github.com/openvinotoolkit/cvat/issues/1143
I couldn't find panoptic json files with colors inside, just additional https://github.com/cocodataset/panopticapi/blob/master/panoptic_coco_categories.json

Reply:
Indeed, there is no color information even in the official annotations. But, actually, it is not used for mask parsing, because `segment_info.id` is enough here, so, maybe, we could avoid requiring a special file. I just want this format to be able to read the unmodified official dataset as a minimum.